src/world.rs

```diff
@@ -413,10 +413,25 @@ impl World {
     }
 
     #[inline(always)]
-    pub(crate) fn entities_meta(&self) -> &[EntityMeta] {
+    pub(crate) fn entities_meta(&self) -> &Vec<EntityMeta> {
         &self.entities.meta
     }
 
+    /// Get the id's of all pending (free) entities
+    pub fn pending(&self) -> &Vec<u32> {
```

Comment:
`World::pending` is a deeply obscure name. How about `World::entity_freelist`? Regardless, the doc comment should explain what this is and why it's useful.

Reply:
Renamed and tried to improve the doc of Freelist.